features/rendering.feature

```diff
@@ -30,6 +30,34 @@ Feature: Rendering
     Then  I should get a non-zero exit-status
     And   I should see "Liquid Exception: Liquid error \(.+/_includes/invalid\.html line 1\): wrong number of arguments (\(given 1, expected 2\)|\(1 for 2\)) included in index\.html" in the build output
 
+  Scenario: Rendering a default site containing a file with rogue Liquid constructs
+    Given I have a "index.html" page with title "Simple Test" that contains "{{ page.title | foobar }}\n\n{{ page.author }}"
+    When  I run jekyll build
+    Then  I should get a zero exit-status
+    And   I should not see "Liquid Exception:" in the build output
+
+  Scenario: Rendering a custom site containing a file with a non-existent Liquid variable
+    Given I have a "index.html" page with title "Simple Test" that contains "{{ page.title }}\n\n{{ page.author }}"
+    And   I have a "_config.yml" file with content:
+    """
+    liquid:
+      strict_variables: true
+    """
+    When  I run jekyll build
+    Then  I should get a non-zero exit-status
+    And   I should see "undefined variable author in index.html" in the build output
```

Comment:
Is a line number printed? Could we test for that?

Reply:
sure..